theseus/core/vectorizer.py

```diff
@@ -33,6 +34,18 @@ def _varinfo(var) -> str:
     )
 
 
+# Identifies what quantities vectorizaton computes for all cost functions
+class _VectorizationMode(Enum):
```

Comment:
Are these three models selected by the user? Looks not?

Reply:
No, they are currently private, as all of this happens under the hood.